ern-core/src/createProxyAgent.ts

```diff
@@ -40,5 +40,9 @@ export function createProxyAgentFromErnConfig(
   const proxyUrl = config.get(configKey)
   if (proxyUrl) {
     return createProxyAgentFromUrl(new url.URL(proxyUrl), { https })
+  } else if (process.env.http_proxy) {
+    return createProxyAgentFromUrl(new url.URL(process.env.http_proxy), { https })
```

Comment:
http?

Reply:
Yeah, I agree it looks odd (it threw me off at first too), but if you check the code further up, `https` is actually a boolean that is passed in to this function. It later on determines which `tunnel` methods to call in `createProxyAgentFromUrl`.